site/src/test/java/uk/nhs/digital/ps/beans/navigation/CiBreadcrumbTest.java

```diff
@@ -0,0 +1,236 @@
+import org.onehippo.forge.breadcrumb.om.BreadcrumbItem;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import uk.nhs.digital.ps.beans.*;
+import uk.nhs.digital.ps.test.util.ReflectionHelper;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RunWith(PowerMockRunner.class)
+@PowerMockIgnore("javax.management.*")
+@PrepareForTest(HstQueryBuilder.class)
+public class CiBreadcrumbTest {
+
+    @Mock private HstRequest hstRequest;
+    @Mock private HstRequestContext hstRequestContext;
+    @Mock private Object currentDocumentBean;
+    @Mock private HippoBean siteBaseBean;
+    @Mock private HstQueryBuilder queryBuilder;
+    @Mock private HstQuery query;
+    @Mock private HstQueryResult queryResult;
+    @Mock private HippoBeanIterator hippoBeanIterator;
+    @Mock private CiLanding ciLandingBean;
+    @Mock private HstLinkCreator hstLinkCreator;
+    @Mock private HstLink hstLink;
+    @Mock private Publication publication;
+    @Mock private Series series;
+
+    private static final String ARBITRARY_PATH = "/some/arbitrary/path/to/document";
+    private static final String TITLE_CI_LANDING = "Clinical Commissioning Group Outcomes Indicator Set (CCG OIS)";
+    private static final String TITLE_ARCHIVE = "Archived CCG Outcomes Indicator Set Publications";
+    private static final String TITLE_SERIES = "Clinical Commissioning Group Outcomes Indicator SERIES";
+    private static final String TITLE_PUBLICATION = "CCG Outcomes Indicator Set - March 2018";
+    private static final String TITLE_DATASET = "1.2 Under 75 mortality from cardiovascular disease";
+
+    @Before
+    public void setUp() throws Exception {
+        initMocks(this);
+
+        // Mock this static method so we can simulate the building and execution of a query
+        PowerMockito.mockStatic(HstQueryBuilder.class);
+        PowerMockito.when(HstQueryBuilder.create(siteBaseBean)).thenReturn(queryBuilder);
+
+        given(queryBuilder.ofTypes(any(Class.class))).willReturn(queryBuilder);
+        given(queryBuilder.where(or(any()))).willReturn(queryBuilder);
+        given(queryBuilder.build()).willReturn(query);
+        given(query.execute()).willReturn(queryResult);
+        given(queryResult.getHippoBeans()).willReturn(hippoBeanIterator);
+
+        ReflectionHelper.callMethod(RequestContextProvider.class, "set", HstRequestContext.class, hstRequestContext);
+
+        given(hstRequestContext.getHstLinkCreator()).willReturn(hstLinkCreator);
+        given(hstLinkCreator.create(ciLandingBean,hstRequestContext)).willReturn(hstLink);
+        given(hstLink.getPath()).willReturn(ARBITRARY_PATH);
+    }
+
+    private void mockCurrentDocumentBean(Class<?> docClass, String docTitle) {
+        currentDocumentBean = mock(docClass);
```

Comment:
Don't think this should be tagged with `@mock` as a variable if you are assigning to a mock manually here. Could also make it of type `BaseDocument` to avoid unnecessary casts